packages/react/src/field/Field.tsx

```diff
@@ -51,6 +63,13 @@ export const Field = forwardRef<HTMLDivElement, FieldProps>(
                   *
                 </Text>
               )}
+              {info && (
+                <Tooltip content={info} side="top">
+                  <Box asChild h="12" ml="2" w="12">
```

Comment:
we can shorten w/h to `size`
```suggestion
                  <Box asChild ml="2" size="12">
```

Reply:
the gap is `2` vai, but for the icon, the gap in figma is `4`, that's why it was added vai.

Reply:
then we can just change the gap to 4 on the parent and remove this `ml` ?

Reply:
Vai, the required asterisk has gap 2 from `label`

Reply:
aiyah - the info icon should not be inside the `<label>` - you need to introduce an additional `<div>` element here wrapping the label and the info icon and add the gap there